src/main/scala/com/campudus/tableaux/database/domain/table.scala

```diff
@@ -3,14 +3,28 @@ package com.campudus.tableaux.database.domain
 import com.campudus.tableaux.database.model.TableauxModel._
 import org.vertx.scala.core.json._
 
-case class Table(id: TableId, name: String, hidden: Boolean) extends DomainObject {
-  override def getJson: JsonObject = Json.obj("id" -> id, "name" -> name, "hidden" -> hidden)
+case class Table(id: TableId, name: String, hidden: Boolean, langtags: Option[Seq[String]]) extends DomainObject {
+  override def getJson: JsonObject = {
+    val result = Json.obj(
+      "id" -> id,
+      "name" -> name,
+      "hidden" -> hidden
+    )
+
+    if (langtags.isDefined) {
+      result.mergeIn(Json.obj("langtags" -> langtags.orNull))
```

Comment:
Shouldn't this be `langtags.get` as you already checked `langtags.isDefined`?
Does `Seq` translate to `JsonArray` in this case?